ibis/backends/dask/execution/arrays.py

```diff
@@ -44,6 +44,12 @@
 )
 
 
+@execute_node.register(ops.ArrayColumn, list)
+def execute_array_column(op, cols, **kwargs):
+    df = dd.concat(cols, axis=1)
+    return df.apply(lambda row: list(row), axis=1, meta=(None, 'object'))
```

Comment:
We should probably return np.array as the element instead of python list

Reply:
you should have to be explict in setting an object array (which is fine), other option is a tuple, which i find ok too